addons/website_form/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,21 @@
+---Models in module 'website_form'---
+
+---Fields in module 'website_form'---
+website_form / ir.model                 / website_form_key (char)       : NEW
+# NOTHING TO DO: new field
+
+---XML records in module 'website_form'---
+NEW ir.model: mail.model_mail_mail
+NEW ir.ui.view: website_form.assets_editor
+NEW ir.ui.view: website_form.assets_tests
+NEW ir.ui.view: website_form.contactus_form
+NEW ir.ui.view: website_form.ir_model_fields_view
+NEW ir.ui.view: website_form.ir_model_view
+NEW ir.ui.view: website_form.register_s_website_form
+NEW ir.ui.view: website_form.remove_external_snippets
+NEW ir.ui.view: website_form.s_website_form
+NEW ir.ui.view: website_form.snippet_options
+# NOTHING TO DO: new xmls
+
+NEW website.page: website_form.contactus_thanks [renamed from website_crm module]
+# NOTHING TO DO: noupdate=0: moved from website_crm to website_form
```

Comment:
This requires some work for not losing custom "Thank you" pages, due to the COW (Create On Write) system. You have to:

- Effectively rename XML-IDs on pre. Although they are noupdate=0, there are some references to the view ID on the COW code, so keeping the same ID will help. This also helps with subsequent inheritance of the regular views.
- Update through a query all the views with the same old `key` field to the new key.

Reply:
Done